extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/FunctionPrivilegesBuilderSuite.scala

```diff
@@ -59,7 +63,7 @@ abstract class FunctionPrivilegesBuilderSuite extends AnyFunSuite
   }
 
   protected val reusedDb: String = getClass.getSimpleName
-  protected val reusedDb2: String = getClass.getSimpleName + "2"
+  protected val reusedDb2: String = getClass.getSimpleName + "_2"
```

Comment:
Is this a necessary change?

Reply:
> Is this a necessary change?

As I use `_` as separator, this is necessary

Reply:
Then you may use `OBJECT_NAME_SEPARATOR` for assembling this.